Surface_mesh_simplification/doc/Surface_mesh_simplification/CGAL/Surface_mesh_simplification/Policies/Edge_collapse/Edge_count_ratio_stop_predicate.h

```diff
@@ -0,0 +1,50 @@
+
+namespace CGAL {
+namespace Surface_mesh_simplification {
+
+/*!
+\ingroup PkgSurfaceMeshSimplificationRef
+
+\cgalModels `StopPredicate`
+
+The class `Edge_count_ratio_stop_predicate` is a model for the `StopPredicate` concept
+which returns `true` when the relation between the initial and current number of edges drops below a certain ratio.
+
+\tparam TriangleMesh is the type of surface mesh being simplified, and must be a model of the `MutableFaceGraph` and `HalfedgeListGraph` concepts.
+
+\sa `CGAL::Surface_mesh_simplification::Edge_count_ratio_stop_predicate<TriangleMesh>`
```

Comment:
```suggestion
\sa `CGAL::Surface_mesh_simplification::Edge_count_stop_predicate<TriangleMesh>`
```

Reply:
Why not the third one?

Reply:
Reasoning was:
- Edge_count is related to Edge_count_ratio because they're both edge-related predicates.
- Face_count_ratio is related to Edge_count_ratio because they're both ratio-related predicates.